install_deps.sh

```diff
@@ -28,32 +28,23 @@ CXX_FLAGS="-std=c++1y $LD_FLAGS"
 mkdir -p deps
 cd deps
 
-git clone "https://github.com/google/protobuf.git" --depth 1 -b "v3.0.0-alpha-1"
+git clone "https://github.com/mockingbirdnest/protobuf"
 pushd protobuf
-git am "../../documentation/Setup Files/protobuf.patch"
 ./autogen.sh
 if [ "$PLATFORM" == "Linux" ]; then
-    ./autogen.sh # Really definitely needs to run twice on Linux for some reason.
+    ./autogen.sh # Really definitely needs to run twice on Ubuntu for some reason.
 fi
-./configure CC=clang CXX=clang++ CXXFLAGS="$CXX_FLAGS" LDFLAGS="$LD_FLAGS"
+./configure CC=clang CXX=clang++ CXXFLAGS="$CXX_FLAGS" LDFLAGS="$LD_FLAGS" LIBS="-lc++ -lc++abi"
 make -j8
 popd
 
-git clone https://github.com/Norgg/glog
+git clone "https://github.com/mockingbirdnest/glog"
 pushd glog
-./configure CC=clang CXX=clang++ CFLAGS="$C_FLAGS" CXXFLAGS="$CXX_FLAGS" LDFLAGS="$LD_FLAGS"
+./configure CC=clang CXX=clang++ CFLAGS="$C_FLAGS" CXXFLAGS="$CXX_FLAGS" LDFLAGS="$LD_FLAGS" LIBS="-lc++ -lc++abi"
 make -j8
 popd
 
-svn checkout http://googletest.googlecode.com/svn/trunk/ gtest
```

Comment:
Same for gmock, that script is pulling our gmock rather than our googlemock at the moment.